lib/pf/cmd/pf/service.pm

```diff
@@ -14,15 +14,11 @@ pfcmd service <service> [start|stop|restart|status|generateconfig|updatesystemd]
 
 Services managed by PacketFence:
 
-  carbon-cache     | carbon-cache daemon
-  carbon-relay     | carbon-relay daemon
-  collectd         | collectd daemon
   haproxy          | haproxy daemon
   httpd.aaa        | Apache AAA webservice
   httpd.admin      | Apache Web admin
   httpd.collector  | Apache Collector daemon
```

Comment:
You need to add the new netdata service + collectdtostatsd

Reply:
done